go/vt/wrangler/traffic_switcher.go

```diff
@@ -1759,3 +1819,43 @@ func reverseName(workflow string) string {
 	}
 	return workflow + reverse
 }
+
+func (ts *trafficSwitcher) addParticipatingTablesToKeyspace(ctx context.Context, keyspace, tableSpecs string) error {
```

Comment:
I _think_ I understand what this function does, from its name; but I'd appreciate a function comment to elaborate a bit.

Reply:
added comment